src/constants/links.js

```diff
@@ -1,5 +1,6 @@
 export const LINKS = {
   home: { path: "/" },
   about: { path: "/about" },
+  hourly: { path: ":dayName" },
   iconURL: { path: "https://openweathermap.org/img/wn/" },
```

Comment:
can we remove iconURL from LINKS and move it to constants like this

export cont iconURL = "https://openweathermap.org/img/wn/"

I know this may be a difficult change todo but it will look more clear because external link should not come inside app route links

Reply:
Done